### PR TITLE
Add subsection on documentation.

### PR DESCRIPTION
**Pull request by rgommers**
This text was proposed via email by Joe Harrington.

Note: just wanted to submit this as a PR before the weekend, so it can be discussed. Haven't had the chance to build it, there may be a couple of minor latex issues.

### CODE REVIEW

File: scipy-1.0/paper.tex [hunk 1]
```diff
@@ -642,6 +642,35 @@ \subsection*{SciPy Matures}
...
+editor.  Pauli Virtanen, Emmanuelle Gouillart, and Gael Varoquaux wrote
+numpydoc, a system for collecting, reviewing, and safely integrating
+documentation with the Python sources.  The system, based on Sphinx,
+produced text, hypertext, and PDF-formatted documents from a single
+source document.  Harrington proposed and they established a
+documentation template and a review system.  Using the SciPy mailing
+list, they solicited community help, and by the end of the summer had
+high-quality documentation for the most frequently used NumPy and SciPy
+routines.  Each contributor of 1000 words or more received a T-shirt
+reading "SciPy Documentation Marathon: I WTFM", emphasizing that the
```
Comment (stefanv) [inline scipy-1.0/paper.tex]: Probaby too much detail for this paper.

File: scipy-1.0/paper.tex [hunk 2]
```diff
@@ -642,6 +642,35 @@ \subsection*{SciPy Matures}
...
+project was no mere coding sprint.  In 2009, van der Walt became a core
+developer and David Goldsmith stepped up as lead editor.  The lead
+editor's job included maintaining discussion and interest on the mailing
+lists, which led to high participation.  By 2010, over 75 people had
+contributed significantly to over 1000 printed pages of reference
+documentation\cite{harrington2008scipy, harrington2009scipy}.
```
Comment (stefanv) [inline scipy-1.0/paper.tex]: Additional reference, for what it's worth: http://conference.scipy.org/proceedings/scipy2008/paper_5/full_text.pdf

**Comment by mdhaber** [comment]
Submitted PR to @rgommers repo: https://github.com/rgommers/scipy-articles/pull/2

File: scipy-1.0/paper.tex [hunk 3]
```diff
@@ -642,6 +642,35 @@ \subsection*{SciPy Matures}
 called PyData.
 
 
+\subsection{*Documentation}
+Early versions of SciPy had minimal documentation.  Travis
+Oliphant wrote \textit{A guide to NumPy}, an 8000 word e-book, which
```
[the remark this answers is not in the document]
Comment (rgommers) [reply scipy-1.0/paper.tex]: hmm no idea, I just took the content from Joe. this seems simply wrong

**Comment by mdhaber** [comment]
@stefanv how does this look?

**Comment by rgommers** [comment]
let's merge in 24 hours if no further comments, so we're ready to submit on the weekend.

**Comment by tylerjereddy** [comment]
Looks ok to me too. I don't see any major LaTeX formatting issues in the rendered text changed here.

In it goes

**Comment by stefanv** [comment]
The current version looks accurate to me and reads well; thank you.

**Pull request merged**